### PR TITLE
`IActionContext.BlockHash` property

### PR DESCRIPTION
**Pull request by dahlia**
Closes <https://github.com/planetarium/libplanet/issues/1295>.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -256,7 +257,12 @@ To be released.
 [#1274]: https://github.com/planetarium/libplanet/pull/1274
 [#1275]: https://github.com/planetarium/libplanet/pull/1275
 [#1278]: https://github.com/planetarium/libplanet/pull/1278
+<<<<<<< HEAD
 [#1289]: https://github.com/planetarium/libplanet/pull/1289
+=======
+[#1295]: https://github.com/planetarium/libplanet/issues/1295
+[#1296]: https://github.com/planetarium/libplanet/pull/1296
+>>>>>>> 9d9bcd87b (IActionContext.BlockHash property)
```
Comment (limebell) [inline CHANGES.md:265]:
```suggestion
[#1289]: https://github.com/planetarium/libplanet/pull/1289
[#1295]: https://github.com/planetarium/libplanet/issues/1295
[#1296]: https://github.com/planetarium/libplanet/pull/1296
```
Conflict sign still exists 👀

**Review by limebell — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (dahlia) [reply CHANGES.md]: Amended.

**Review by area363 — APPROVED** [review]
🙇

**Review by limebell — APPROVED** [review]

**Review by area363 — APPROVED** [review]

**Comment by dahlia** [comment]
@planetarium/libplanet Please take a look again.

**Review by longfin — APPROVED** [review]

**Pull request merged**